src/components/ReportActionItem/ExportWithDropdownMenu.tsx

```diff
@@ -0,0 +1,123 @@
+import React, {useCallback, useMemo, useState} from 'react';
+import {useOnyx} from 'react-native-onyx';
+import type {OnyxEntry} from 'react-native-onyx';
+import type {ValueOf} from 'type-fest';
+import ButtonWithDropdownMenu from '@components/ButtonWithDropdownMenu';
+import type {DropdownOption, ReportExportType} from '@components/ButtonWithDropdownMenu/types';
+import ConfirmModal from '@components/ConfirmModal';
+import useLocalize from '@hooks/useLocalize';
+import useResponsiveLayout from '@hooks/useResponsiveLayout';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as ReportActions from '@libs/actions/Report';
+import * as ReportUtils from '@libs/ReportUtils';
+import type {ModalStatus} from '@pages/home/report/ReportDetailsExportPage';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {Report} from '@src/types/onyx';
+
+type ExportWithDropdownMenuProps = {
+    report: OnyxEntry<Report>;
+
+    integrationName: ValueOf<typeof CONST.POLICY.CONNECTIONS.NAME>;
```

Comment:
We already have such type in 'src/types/onyx/Policy.ts'
```suggestion
    integrationName: ConnectionName;
```